internal/lookout/repository/jobs.go

```diff
@@ -280,11 +286,9 @@ func makeJobFromRow(row *JobRow) (*api.Job, error) {
 		return nil, nil
 	}
 
-	var jobFromJson api.Job
-	jobJson := ParseNullString(row.JobJson)
-	err := json.Unmarshal([]byte(jobJson), &jobFromJson)
+	jobSpec, err := makeApiJob([]byte(row.OrigJobSpec.String))
 	if err != nil {
-		log.Errorf("error while parsing job %s json: %v", ParseNullString(row.JobId), err)
+		log.Warnf("unable to load orig job spec. %+v", err)
```

Comment:
Is this temporary?

Reply:
Error -> Warn I mean?

Reply:
not sure why i changed that, will switch back